src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -266,16 +267,24 @@ struct
       ; transition_registry: Transition_registry.t
       ; best_tip_diff: Best_tip_diff.t
       ; root_diff: Root_diff.t
-      ; persistence_diff: Persistence_diff.t }
+      ; persistence_diff: Persistence_diff.t
+      ; new_transition:
+          Inputs.External_transition.Verified.t New_transition.Var.t }
```

Comment:
@nholland94 was saying this will be refactored soon, but for now I think this'll work.

Reply:
Yes, hoping to refactor this in the next week or 2.